app/routers/weight.py

```diff
@@ -0,0 +1,67 @@
+from typing import Union
+
+from fastapi import APIRouter, Depends, Request
+from sqlalchemy.orm import Session
+from starlette.responses import RedirectResponse
+
+from app.database.models import User
+from app.dependencies import get_db
+from app.dependencies import templates
+
+
+router = APIRouter(tags=["weight"],)
+
+
+@router.get("/weight")
+async def get_weight(
+        request: Request,
+        session: Session = Depends(get_db),
+        target: Union[float, None] = None,
+        current_weight: Union[float, None] = None,
+        ):
+
+    # TODO Waiting for user registration
+    user = session.query(User).filter_by(id=1).first()
+    target = user.target_weight
+    if current_weight:
+        return RedirectResponse(url='/')
+    return templates.TemplateResponse("weight.html", {
+        "request": request,
+        "target": target,
+        "current_weight": current_weight,
+        }
+    )
+
+
+@router.post("/weight")
+async def weight(
+        request: Request,
+        session: Session = Depends(get_db)):
+    user = session.query(User).filter_by(id=1).first()
+    data = await request.form()
+    target = data['target']
+    current_weight = data['current_weight']
+    if target:
+        user.target_weight = target
+        session.commit()
+    else:
+        target = user.target_weight
+        if not target:
+            target = current_weight
+    way_to_go = float(current_weight) - float(target)
+    way_to_go = round(way_to_go, 2)
```

Comment:
why is it always a float?

Reply:
To allow float numbers and float weight difference